src/lib/gundb/UserStorageClass.js

```diff
@@ -105,7 +105,7 @@ export type TransactionEvent = FeedEvent & {
 }
 
 export const welcomeMessage = {
-  id: '0',
+  id: '1',
```

Comment:
Why?

Reply:
Due to duplicate IDs, tests did not pass as the card was overwritten

Reply:
Card 'Invite Friends' has id = 0